### PR TITLE
feat(content): more clouds, changes to weather behavior

### PR DESCRIPTION
**Pull request by VissValdyr**
## Purpose of change
Building on top of @chaosvolt PR #4515 .
Weather got better but was not quite right, it stayed cloudless a lot of times, so I digged the file and used the "either" rule for baro and hydro not "and". The weather now is a lot more likely to change to cloudy, and so changing to rainy. Same with sunny, it was rare, so it also got the "either" treatment.

Light drizzle is a rain type, so instead of light drizzle, it will flurry.

## Describe the solution
1. Sunny and cloudy get "either" not "and" for baro and hydro.
2. Values got changed, so it is not always cloudy or sunny.
3. Renamed "Clear" to "Fair"

## Testing
Works fine. It will probably need some more fine tuning in the future. (Less cloudy or less rainy, or even the opposite)

## Additional context
Screenshot_20240429-221346_Cataclysm BN (experimental)
Screenshot_20240429-221405_Cataclysm BN (experimental)

### CODE REVIEW

**Comment by VissValdyr** [comment]
Here is the direct comparison:
Old settings:
Screenshot_20240429-231005_Cataclysm BN (experimental)
New settings, same position, same time.
Screenshot_20240429-231527_Cataclysm BN (experimental)

**Review by chaosvolt — APPROVED** [review]

File: data/json/weather_type.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   {
     "id": "clear",
     "type": "weather_type",
-    "name": "Clear",
+    "name": "Fair",
```
Comment (scarf005) [inline data/json/weather_type.json:22]: could you elaborate on why was it renamed from `clear` to `fair`?

Comment (VissValdyr) [reply data/json/weather_type.json:22]: Clear would mean no clouds. But we have sunny, that is when clear weather is present.

Fair means bit of scattered clouds here and there. But not cloudy. Sun is there but not always.

Comment (VissValdyr) [reply data/json/weather_type.json:22]: https://forecast.weather.gov/glossary.php?word=FAIR

Comment (scarf005) [reply data/json/weather_type.json:22]: i see. was confused because there were no explanation on PR description.

Comment (VissValdyr) [reply data/json/weather_type.json:22]: > i see. was confused because there were no explanation on PR description.

Oh, you are right, my bad.

**Comment by VissValdyr** [comment]
@chaosvolt Look at that. Thunderstorm also spawning. A lot of cloudy and rainy days. The solar panels aren't that OP anymore.
Screenshot_20240430-144056_Cataclysm BN (experimental)

**Pull request merged**